input/fsh/modules/files.fsh

```diff
@@ -120,9 +120,9 @@ Title: "NCPI File"
 Description: "Information about a file related to a research participant"
 * ^version = "0.0.1"
 * ^status = #draft
-* identifier 0..* /*File External ID*/
+* identifier 0..1 /*File External ID*/

```

Comment:
I actually don't think we want to restrict identifier to 1 or zero. On the contrary, I think we should limit it to 1..* and require at least one ID because there are good reasons for a file to have more than one useful identifier. There may be only one External ID, but they should be able to put other ones there.